_data/quickstart.yaml

```diff
@@ -11,3 +11,7 @@ toc:
    - quickstart/aws-ec2.md
    - quickstart/aws-extract-thumbnail.md
    - quickstart/aws-s3-website.md
+- title: Kubernetes Tutorials
+  path: /quickstart/kubernetes
+  section:
+  - quickstart/kubernetes-guestbook.md
```

Comment:
Indentation here is slightly different - I assume it doesn't matter - but probably worth being consistent (and who knows with YAML what will cause things to mis-parse).

Reply:
Good catch, will fix :+1: